libextract/quantifiers.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+"""
+A submodule providing 'quantifying' functions.
+
+The protocol that's followed is each function expects a "node"
+of type lxml.html.HtmlElement (sometimes *._ElementTree), and
+the function will return either a numerical type or a Counter
+where the key=<node> and value=<numerical>
+"""
+
+from libextract.coretools import Counter
+
+
+def text_length(node):
+    """
+    Returns the length of the text contained within
+    a given *node*.
+    """
+    text = node.text
+    return len(' '.join(text.split())) if text else 0
+
+
+def children_counter(node):
```

Comment:
Partly my fault- Better to name it in the imperative. For example: ``count_children``.